v4-client-py/examples/short_term_composite_example.py

```diff
@@ -0,0 +1,78 @@
+'''Example for trading with human readable numbers
+
+Usage: python -m examples.composite_example
+'''
+import asyncio
+import logging
+from random import randrange
+from v4_client_py.chain.aerial.wallet import LocalWallet
+from v4_client_py.clients import CompositeClient, Subaccount
+from v4_client_py.clients.constants import BECH32_PREFIX, Network
+
+from v4_client_py.clients.helpers.chain_helpers import (
+    Order_TimeInForce,
+    OrderType, 
+    OrderSide, 
+    OrderTimeInForce, 
+    OrderExecution,
+)
+from examples.utils import loadJson
+
+from tests.constants import DYDX_TEST_MNEMONIC
+
+
+async def main() -> None:
+    wallet = LocalWallet.from_mnemonic(DYDX_TEST_MNEMONIC, BECH32_PREFIX)
+    network = Network.staging()
+    client = CompositeClient(
+        network,
+    )
+    subaccount = Subaccount(wallet, 0)
+    ordersParams = loadJson('human_readable_short_term_orders.json')
+    for orderParams in ordersParams:
+        side = OrderSide[orderParams["side"]]
+
+        # Get the expiration block.
+        current_block = client.get_current_block()
+        next_valid_block_height = current_block + 1
+        # Note, you can change this to any number between `next_valid_block_height` to `next_valid_block_height + SHORT_BLOCK_WINDOW`
+        good_til_block = next_valid_block_height + 3
+
+        time_in_force = orderExecutionToTimeInForce(orderParams['timeInForce'])
+
+        price = orderParams.get("price", 1350)
+        try:
+            tx = client.place_short_term_order(
+                subaccount,
+                market='ETH-USD',
+                side=side,
+                price=price,
+                size=0.01,
+                client_id=randrange(0, 100000000),
```

Comment:
Suggestion: should this be a random number between 0 and 2^32-1? Maybe note in the comment it's a uint32.